app/plugins/views.plugin.js

```diff
@@ -100,6 +101,9 @@ function context (request) {
       scope: request.auth.credentials?.scope,
       permission: request.auth.credentials?.permission
     },
+    featureFlag: {
```

Comment:
This would/could be a handy place to put any feature flag so there could be more. So, I'd suggest

```suggestion
    featureFlags: {
```